static/app/components/alert.tsx

```diff
@@ -52,6 +59,8 @@ function Alert({
         return <IconCheckmark />;
       case 'error':
         return <IconNot />;
+      case 'performanceError':
+        return <IconFire />;
```

Comment:
To prevent the fire icon to be used for all error alerts in the app (rather then just the event details page), opted to created a new alert type.